### DISCUSSION TITLE
Is the QLMarkdown generate inline CSS for every file preview?

### DISCUSSION THREAD

**Discussion by setanarut** [sbarex/QLMarkdown#45] [Q&A] [answered]
- Is the QLMarkdown generate inline CSS for every file preview?
- Doesn't it make more sense to use the external file with a single line?` <link rel="stylesheet" href="markdown.css">` .
- Does this provide faster viewing?

<img width="399" alt="Ekran Resmi 2021-01-04 00 17 11" src="https://user-images.githubusercontent.com/36481442/103489127-38e0a380-4e23-11eb-816d-11d075bcf175.png">

**Comment by sbarex** [comment, marked as answer]
Yes the app embed the style on the html code.
I chose this mode to have no potential problems loading external files from the quicklook extension. It's a local file so I don't think it would be cached by webkit or would make loading faster if put in a `<link>` tag.
